.eslintignore

```diff
@@ -0,0 +1,7 @@
+node-modules/
+actions/notify-workflow-completed/node_modules/
+actions/notify-workflow-completed/dist/
+
+# Following directories have their own formatting configuration:
+actions/run-workflow/
+actions/upstream-builds-query/
```

Comment:
RE: https://github.com/keep-network/ci/pull/11#discussion_r676604189

We do have a lint script configured for `notify-workflow-completed` action: https://github.com/keep-network/ci/blob/dc09bd92de5c26da1b4331f95efee0af30ff2741/actions/notify-workflow-completed/package.json#L7 
If we're missing any configuration there we should handle it in a separate PR.

```suggestion
node-modules/

# Following directories have their own formatting configuration:
actions/notify-workflow-completed/
actions/run-workflow/
actions/upstream-builds-query/
```